Mathlib/SetTheory/Game/Basic.lean

```diff
@@ -225,6 +228,13 @@ namespace PGame
 @[simp] theorem quot_add (a b : PGame) : ⟦a + b⟧ = (⟦a⟧ : Game) + ⟦b⟧ := rfl
 @[simp] theorem quot_sub (a b : PGame) : ⟦a - b⟧ = (⟦a⟧ : Game) - ⟦b⟧ := rfl
 
+@[simp]
+theorem quot_natCast : ∀ n : ℕ, ⟦(n : PGame)⟧ = (n : Game)
+  | 0 => rfl
+  | n + 1 => by
+    rw [PGame.nat_succ, quot_add, Nat.cast_add, Nat.cast_one, quot_natCast]
+    rfl
```

Comment:
You'd do better to replace the `natCast` field in the `AddCommGroupWithOne` instance above. I think your two proofs here become the `natCast_zero` and `natCast_succ` fields, and then this proof becomes `rfl`.

Reply:
I'd rather not tweak the `NatCast` instances at this point, since there's an issue with the `PGame` instance that needs to be addressed.

The `NatCast` instance on `PGame` doesn't match the actual mathematical definition. It's directly defined as `0 + 1 + 1 + ...`, which due to how game addition works, leads to it having a left set of different relabellings of `n - 1` rather than the expected `0` through `n - 1`. 

The actual definition is given by `Ordinal.toPGame` on a natural argument. But replacing it means `(0 : N)` is no longer equal (not even prop-eq) to `0`. That is, unless, we defined it near the start of the file. Which feels like a messy solution to me.

I made a thread about this on Zulip in the #matlib4 channel, can't link it on mobile unfortunately.

Reply:
Here's the thread: https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Nat.20.E2.86.92.20PGame.20cast.2C.20again/near/469290453

Reply:
The conclusion from that thread (or at least what I concluded) was that it's probably better to wait for `IGame` before tweaking any of these instances.